File: icna-register/icna-register-api/src/main/java/org/icna/register/mapper/EventMapper.java
package org.icna.register.mapper;

import org.icna.register.dto.EventDto;
import org.icna.register.entity.Event;
import org.mapstruct.Mapper;

@Mapper(componentModel = "spring")
public interface EventMapper {
    EventDto beanToDto(Event bean);

    EventDto dtoToBean(EventDto dto);
}

File: icna-register/icna-register-api/src/main/java/org/icna/register/mapper/EventMapper.kt
package org.icna.register.mapper

import org.icna.register.dto.EventDto
import org.icna.register.entity.Event
import org.mapstruct.Mapper

@Mapper(componentModel = "spring")
interface EventMapper {
    fun beanToDto(bean: Event): EventDto
    fun dtoToBean(dto: EventDto): EventDto
}